src/app/compare/compare-tree/compare-tree.component.css

```diff
@@ -1,3 +1,18 @@
 #tree-container {
   background: linear-gradient(#e9ecef, #e9ecef) no-repeat center/2px 100%;
 }
+
+.compare-header {
+  display: flex;
+  align-items: center;
+  height: 5vh;
```

Comment:
Als je het scherm nu minder hoog maakt, blijft de header en zijn componenten dan wel goed zichtbaar?
Misschien is een vaste hoogte beter? 64px of 1em bij wijze van.